src/transforms/coercer.rs

```diff
@@ -42,10 +42,11 @@ impl Transform for Coercer {
                 match conv.convert(value) {
                     Ok(converted) => log.insert_explicit(field.into(), converted),
                     Err(error) => {
-                        debug!(
+                        warn!(
```

Comment:
👍 I believe this transform was merged before we had rate limiting. Unfortunately, https://github.com/timberio/vector/issues/628 didn't cover this transform. I'll do another pass to make sure we're not missing anything new.